README.md

```diff
@@ -46,6 +46,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Premium Minds](https://www.premium-minds.com/) [:rocket:](https://join.premium-minds.com/) | We build great software for innovative clients. | `Lisboa` |
 | [Prime IT](https://www.primeit.pt/) [:rocket:](https://www.primeit.pt/carreiras) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
 | [Redlight Software](https://weareredlight.com) | Web and mobile development studio focused on products. | `Coimbra` `Lisboa` |
+| [Runtime Revolution](https://www.runtime-revolution.com/) [:rocket:](https://www.runtime-revolution.com/careers) | Ruby on Rails, Python, Node.js, React, mobile and product development experts. | `Lisboa` |
```

Comment:
Hi @ivobenedito, can you please shorten the description so that it fits is a single line to improve readability? Thanks!